### PR TITLE
Rename MAC and PHY fields in console

### LINKED ISSUE #2424: Rename PHY and MAC in console
#### Summary

It's unclear for newbies what MAC and PHY are in the console. The LoRaWAN terms for these are (i think)
MAC: LoRaWAN specification or version
PHY: Regional parameters

#### Why do we need this ?

To make it easy to add devices in the v3 console (these terms are also unfamiliar for those upgrading from v2, where they were not required)

#### What is already there? What do you see now?

Screenshot 2020-04-28 at 12 47 28

#### What is missing? What do you want to see?

MAC and PHY replaced with terms used by LoRa Alliance / other device documentation

Screenshot 2020-04-28 at 12 49 49

#### How do you propose to document this?

This is a change to documentation

#### Can you do this yourself and submit a Pull Request?

No, @kschiffer @pgalic96 ?

@htdvisser @johanstokking can you double check my LoRa terminology

### PR DESCRIPTION
**Pull request by pgalic96**
#### Summary

Closes #2424 

#### Changes

- Change `MAC version` to `LoRaWAN version` and `PHY version` to `Regional Parameters version` 

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/lib/shared-messages.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ export default defineMessages({
   payloadFormattersUpdateFailure: 'There was an error updating the payload formatter',
   payloadFormattersUpdateSuccess: 'Payload formatter updated',
   payloadFormattersUplink: 'Uplink payload formatters',
-  phyVersion: 'PHY version',
+  phyVersion: 'Regional Parameters version',
```
Comment (kschiffer) [inline pkg/webui/lib/shared-messages.js:190]: I'm a bit undecided whether we should change the message ID as well to something like `regionalParametersVersion`, but I think it's fine.

Comment (kschiffer) [reply pkg/webui/lib/shared-messages.js:190]:
```suggestion
  phyVersion: 'Regional parameters version',
```
Sentence case.

Comment (pgalic96) [reply pkg/webui/lib/shared-messages.js:190]: yeah I agree it is fine, as this is mostly for the user clarity

**Comment by pgalic96** [comment]
@benolayinka can you think of something for https://github.com/TheThingsNetwork/lorawan-stack/pull/2451#pullrequestreview-403274683 ?

File: pkg/webui/lib/shared-messages.js (hunk 1 above)
Comment (johanstokking) [reply pkg/webui/lib/shared-messages.js]: "Regional Parameters" are a spec, so that should be title case

**Comment by benolayinka** [comment]
> What about the field description? @benolayinka can you make a proposal for rewording this as well?
> 
> image
> (talking about the message under the field)
> 
> Also @pgalic96 please always post a screenshot for all frontend PRs with visual changes

I'd say
`The LoRaWAN version (MAC), as provided by the device manufacturer.`
`The Regional Parameters version (PHY), as provided by the device manufacturer.`

All LoRaWAN devices are backwards compatible with the v2 defaults `MAC_V1_0_2` and
`MAC_PHY_V1_0_2_REV_B`, right @johanstokking ? If so, it could be helpful to add that in the description as well

`The LoRaWAN version (MAC), as provided by the device manufacturer. All LoRaWAN devices are backwards compatible with 1.0.2`

or to add it as a note/default selection

`MAC V1.0.2 (Compatible with all end devices)`

**Comment by johanstokking** [comment]
> All LoRaWAN devices are backwards compatible with the v2 defaults `MAC_V1_0_2` and
> `MAC_PHY_V1_0_2_REV_B`, right @johanstokking ? If so, it could be helpful to add that in the description as well

No, we can't make this assumption about version compatibility.

**Comment by johanstokking** [comment]
Please hold merge until #2488 is merged

**Pull request merged**